src/roulette.rs

```diff
@@ -1,6 +1,19 @@
 use lazy_static::lazy_static;
 use rand::Rng;
-use serde::Deserialize;
+use serde::{de::DeserializeOwned, Deserialize};
+
+enum JsonFile {
```

Comment:
Maybe. Since Rust actually validates the existence of the file that would work fine. At the same time it doesn't complicate much IMO, and it would make it easier if we wanted to rename or add more JSON files.